index.html

```diff
@@ -1,14 +1,101 @@
 <!doctype html>
 <html lang="en">
   <head>
+    <!-- Required meta tags -->
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="stylesheet" href="assets/css/starter.css">
-    <title>Bootstrap npm starter</title>
+    <!-- Bootstrap CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6" crossorigin="anonymous">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Condensed:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://use.fontawesome.com/f8a3ee0e31.js"></script>
+    <title>Newsweek</title>
   </head>
-  <body>
-    <h1>Hello world!</h1>
+  <body class="d-flex flex-column min-vh-100">
+    <header class="row">
+      <nav class="d-flex justify-content-between top-nav">
+        <h3 class="d-none d-lg-inline align-self-end">Wed, Jan 20, 2021</h3>
+        <div class="brand d-flex align-self-end">
+          <a href="#"><img class="align-self-center" src="https://upload.wikimedia.org/wikipedia/commons/thumb/d/db/Newsweek_Logo.svg/800px-Newsweek_Logo.svg.png" alt="logo"></a>
+        </div>
+        <div class="mobile-right flex-last d-flex align-items-center align-self-end">
+          <i class="fa fa-user user"></i>
+          <h2 class="d-none d-md-inline"><a href="#">LOGIN</a></h2>
+          <button type="button" class="btn btn-warning">SUBSCRIBE</button>
+          <i class="fa fa-bars humberger d-lg-none"></i>
+        </div>
+      </nav>
+      <nav class="d-none d-lg-flex nav-2 align-content-center border-bottom">
+        <h4 class="border-end flex-fill"><a href="#">U.S </a></h4>
+        <h4 class="border-end flex-fill"><a href="#">World</a></h4>
+        <h4 class="border-end flex-fill"> <a href="#">Business</a></h4>
+        <h4 class="border-end flex-fill"> <a href="#">Tech & Science</a></h4>
+        <h4 class="border-end flex-fill"> <a href="#">Culture</a></h4>
+        <h4 class="border-end flex-fill"> <a href="#">Newsgeek</a></h4>
+        <h4 class="border-end flex-fill"> <a href="#">Sports</a></h4>
+        <h4 class="border-end flex-fill"> <a href="#">Health</a></h4>
+        <h4 class="border-end flex-fill"> <a href="#">The Debate</a></h4>
+        <h4 class="border-end flex-fill"> <a href="#">Experts</a></h4>
+        <h4 class="ps-3"> <a href="#">Vantage</a></h4>
+        <form class="search flex-fill">
+          <h6>Search <i class="fa fa-search"></i></h6>
+        </form>
```

Comment:
Hint: you can do like this. but remember to style so that it looks the same.
 `<input type="text" placeholder="Search>` 

To keep the icon inside you can use the  `position` property.

Reply:
Thank you Dipeshtwis for that hint.